charts/graph-network-indexer/templates/indexer-agent/statefulset.yaml

```diff
@@ -51,42 +52,51 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
-          {{- with (deepCopy $.Values.indexerDefaults.env | mustMerge $values.env) }}
-          {{- range $key, $val := .}}
-          {{- if not (kindIs "invalid" $val) }}
+          {{- if $values.env }}
+          {{- range $key, $val := $values.env }}
             - name: {{ $key | quote }}
               value: {{ $val | quote }}
           {{- end }}
           {{- end }}
-          {{- end }}
-          {{- with (deepCopy $.Values.indexerDefaults.secretEnv | mustMerge $values.secretEnv) }}
-          {{- range $key, $val := .}}
+          {{- if $values.secretEnv }}
+          {{- range $key, $val := $values.secretEnv }}
             - name: {{ $key | quote }}
               valueFrom:
                 secretKeyRef:
                   name: {{ $val.secretName | quote }}
                   key: {{ $val.key | quote }}
-                  optional: false
           {{- end }}
           {{- end }}
           args:
-          {{- with (deepCopy $.Values.indexerDefaults.config | mustMerge $values.config) }}
-          {{- range $key, $val := .}}
-          {{- if not (kindIs "invalid" $val) }}
-            - --{{ $key }}={{ $val }}
-          {{- end }}
-          {{- end }}
-          {{- end }}
+            - --graph-node-query-endpoint={{ index $values.config.graph_node "query_url" }}
+            - --graph-node-status-endpoint={{ index $values.config.graph_node "status_url" }}
+            - --indexer-address={{ index $values.config.indexer "indexer_address" }}
+            {{- with $values.config.subgraphs }}
+            {{- if hasKey .network "query_url" }}
+            - --network-subgraph-endpoint={{ index .network "query_url" }}
+            {{- end }}
+            {{- if hasKey .network "deployment_id" }}
+            - --network-subgraph-deployment={{ index .network "deployment_id" }}
+            {{- end }}
+            {{- if hasKey .escrow "query_url" }}
+            - --tap-subgraph-endpoint={{ index .escrow "query_url" }}
+            {{- end }}
```

Comment:
the keys I see in values are named "sugraphs.network" and "subgraphs.escrow",
```yaml
subgraphs.network:
subgraphs.escrow:
```

the logic here seems to be designed for:
```yaml
subgraphs:
  network:
  escrow:
```